_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,13 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added Date picker usability guidance.
+    summaryAdditional: Users should always be able to enter dates manually in Date picker.
```

Comment:
**suggestion:** Let's be more specific about the type of guidance added. Just want to make it easy to immediately understand what was changed. Here is a possible suggestion (feel free to edit): 
```suggestion
    summary: Added guidance about always allowing users to type in the date manually.
```

Note: We should update both the date picker and date range picker changelogs with the same summary.